### PR TITLE
Update integers-and-floating-point-numbers.md

### PR DESCRIPTION
**Pull request by annainfo**
Make text style more scientific, replace "we"

### CODE REVIEW

File: doc/src/manual/integers-and-floating-point-numbers.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ UInt128: [0,340282366920938463463374607431768211455]
 ```
 
 The values returned by [`typemin`](@ref) and [`typemax`](@ref) are always of the given argument
-type. (The above expression uses several features we have yet to introduce, including [for loops](@ref man-loops),
+type. (The above expression uses several features that yet have to be introduced, including [for loops](@ref man-loops),
```
Comment (ViralBShah) [inline doc/src/manual/integers-and-floating-point-numbers.md]: I am not sure this is necessarily better. But the sentence construction is weird. Should be "features that have yet to be introduced".

**Review by ViralBShah — CHANGES_REQUESTED** [review]

File: doc/src/manual/integers-and-floating-point-numbers.md (hunk 1 above)
Comment (annainfo) [reply doc/src/manual/integers-and-floating-point-numbers.md]: Agree with changing to "that have yet to be introduced" (Google gives "that yet have to": 79K, "that+have+yet+to" 25M)

Re removing "we":

Precedence: The most seen style for descriptive text in English might be the one from Wikipedia. The encyclopedic articles don't contain "we" outside citations. 

Logic: Technically, "we" in the Julia manual is undefined. Who is "we"? Furthermore, does the reader care about that those behind "we" introduce the features, or is it sufficient that the features are introduced? If the latter is true, then "we" is superfluous.

Comment (cormullion) [reply doc/src/manual/integers-and-floating-point-numbers.md]: There are arguments in favour of a less passive voice, too. See [this page](https://math.stackexchange.com/questions/668645/using-we-have-in-maths-papers) for some interesting aspects to the discussion:

> However, the pronoun “we” is now generally considered acceptable in contexts where it means the author and reader together, or less often, the author with the reader looking on.

Comment (annainfo) [reply doc/src/manual/integers-and-floating-point-numbers.md]: The place you link to, claims the citation comes from http://www.math.uchicago.edu/~may/VIGRE/VIGRE2010/piiUJM2up.pdf

You mean that is more authoritative than the style of the English Wikipedia?

Re "There are arguments in favour of a less passive voice, too." - Which?

**Review by StefanKarpinski — APPROVED** [review]

**Comment by StefanKarpinski** [comment]
Thank you for the doc PR!

**Pull request merged**